### DISCUSSION TITLE
Multi-select with addition of options

### DISCUSSION THREAD

**Discussion by sbettid** [carbon-design-system/carbon#17314] [Help]
Hi everyone! 

I would kindly ask y'all for a feedback, if you have a couple minutes time. 

When working on the UI of one of our project, we face the need of having a filterable multi-select that enables both the viewing of the selected options and also the possibility to add new options to the existing ones. 

I see that the Carbon component allows to filter the options but they are visualized as selected only in the dropdown and not, for example, in the main field (and for us this would be quite handy), while the addition of new options is not available out of the box. 

Can I ask you why you think this is not present and also how would you tackle this, from a design perspective? 

Using other components, such as an "Add" button near to the multi-select or are there other options?

Thank you in advance for your time and help!

**Comment by adamalston** [comment]
Hi, are you looking for something like https://github.com/carbon-design-system/carbon/issues/10345?
